tests/svtypes/typedef_initial_and_assign.ys

```diff
@@ -0,0 +1,6 @@
+logger -expect warning "reg '\\var_3' is assigned in a continuous assignment" 1
```

Comment:
Please add `logger -expect-no-warnings` so that the script will error if are any warnings besides those explicitly expected.